spec/jobs/segment_track_job_spec.rb

```diff
@@ -57,7 +59,7 @@
         stub_const('ENV', 'LAGO_DISABLE_SEGMENT' => 'true')
 
         expect(SEGMENT_CLIENT).not_to receive(:track)
-        subject.perform_now(membership_id: membership_id, event: event, properties: properties)
+        subject.perform_now(membership_id:, event:, properties:)
```

Comment:
RSpec/NamedSubject: Name your test subject if you need to reference it explicitly. (https://rspec.rubystyle.guide/#use-subject, https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/NamedSubject)